locales/es/messages.jsonc

```diff
@@ -0,0 +1,1433 @@
+      },
+      "homeheader_secretmessage_Default6": {
+            "message": "focas"
+      },
+      "homeheader_secretmessage_Default7": {
+            "message": "Adiós, LegacyApi!"
+      },
+      "homeheader_secretmessage_Default8": {
+            "message": "Deja de faltar a la escuela."
+      },
+      "homeheader_secretmessage_Default9": {
+            "message": "Boicotea las malas experiencias, rechaza las experiencias pirateadas."
+      },
+      "homeheader_secretmessage_Default10": {
+            "message": "Sé consciente de la autodefensa y de ser engañado."
+      },
+      "homeheader_secretmessage_Default11": {
+            "message": "Jugar es bueno para tu cerebro, pero jugar demasiado puede perjudicar tu salud."
+      },
+      "homeheader_secretmessage_Default12": {
+            "message": "Administra bien tu tiempo y disfruta de un estilo de vida saludable."
+      },
+      "homeheader_secretmessage_Default13": {
+            "message": "Imagina 1M de Robux..."
+      },
+      "homeheader_secretmessage_Default14": {
+            "message": "¿Cómo estás?"
+      },
+      "homeheader_secretmessage_Default15": {
+            "message": "Esperamos que estés disfrutando de RoSeal."
+      },
+      "homeheader_secretmessage_Default16": {
+            "message": "Jejeje..."
+      },
+      "homeheader_secretmessage_Default17": {
+            "message": "Todo en Roblox sucede detrás de los ojos de todos."
+      },
+      "homeheader_secretmessage_Default18": {
+            "message": "[ Contenido Eliminado ]"
+      },
+      "homeheader_secretmessage_Default19": {
+            "message": "Este no es un mensaje de prueba."
+      },
+      "homeheader_secretmessage_Default20": {
+            "message": "gatito."
+      },
+      "homeheader_secretmessage_Default21": {
+            "message": "pájaro."
+      },
+      "homeheader_secretmessage_Default22": {
+            "message": "Inserta una imagen de foca aquí"
+      },
+      "homeheader_secretmessage_Default23": {
+            "message": "Inserta una imagen de un gatito aqui"
+      },
+      "homeheader_secretmessage_Default24": {
+            "message": "Inserta una imagen de un pajaro aqui"
+      },
+      "homeheader_secretmessage_Default25": {
+            "message": "Roblox caido?"
+      },
+      "homeheader_secretmessage_Default26": {
+            "message": "David Baszucki"
+      },
+      "homeheader_secretmessage_Default27": {
+            "message": "Recuerdas los guest?"
+      },
+      "homeheader_secretmessage_Default28": {
+            "message": "Recuerdas los foros?"
+      },
+      "homeheader_secretmessage_Default29": {
+            "message": "Hola!"
+      },
+      "homeheader_secretmessage_Default30": {
+            "message": "um... yum?"
+      },
+
+      // Added by Furrycality with Love UwU 
+      "homeheader_secretmessage_Default31": {
+            "message": "Howis!! Furrycality tradujo esto UwU"

```

Comment:
this wouldn't exactly work right now, but i might consider adding a secret message specifically for non-english locales as a different name, potentially `homeheader_secretmessage_NonEnglish`, i don't know

Reply:
I am agree